app/src/main/java/org/simple/clinic/home/report/ReportsScreen.kt

```diff
@@ -45,6 +47,23 @@ class ReportsScreen : BaseScreen<
     )
   }
 
+  override fun defaultModel() = ReportsModel.create()
+
+  override fun bindView(layoutInflater: LayoutInflater, container: ViewGroup?) =
+      ScreenReportBinding.inflate(layoutInflater, container, false)
+
+  override fun uiRenderer() = ReportsUiRenderer(this)
+
+  override fun events() = Observable
+      .mergeArray(webViewBackClicks)
```

Comment:
Can we also report events to analytics here? I think we might have forgotten to do that in the original screen implementation?